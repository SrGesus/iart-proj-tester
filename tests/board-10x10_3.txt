FC	BE	FE	FD	VC	LV	LH	BD	VC	FD
VD	VD	FC	BC	BD	LH	FD	LH	VC	VC
BE	FE	FC	BC	BD	VE	LH	VC	FE	VE
LH	FE	BE	BE	LV	VC	LH	LV	BB	VB
LH	BE	BD	FC	BB	LV	LH	FD	VD	FB
LV	LH	VD	FD	VC	LV	FD	VB	LH	VC
VC	BE	BC	LV	BE	BD	BB	VB	VE	BD
FD	BD	VD	FD	LH	LV	VC	FE	LH	FD
VD	BE	FD	VE	BD	VC	BE	FD	VB	VE
VE	LH	LH	FE	FC	FB	BC	FC	FB	VD
