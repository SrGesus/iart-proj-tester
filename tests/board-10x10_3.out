FD	BB	FE	FB	VB	LH	LH	BB	VE	FB
VB	VC	FD	BE	BD	LH	FE	LV	VD	VC
BD	FE	FB	BD	BE	VB	LH	VC	FD	VE
LV	FB	BD	BE	LV	VD	LH	LH	BB	VC
LV	BD	BE	FC	BD	LH	LH	FE	VD	FE
LV	LV	VD	FE	VD	LH	FE	VB	LH	VE
VD	BC	BB	LH	BB	BB	BB	VC	VB	BE
FD	BB	VC	FB	LV	LV	VD	FE	LV	FC
VB	BC	FE	VD	BE	VD	BB	FE	VD	VE
VD	LH	LH	FE	FC	FD	BC	FE	FD	VC
